test/jdk/java/nio/channels/FileChannel/Transfer2GPlus.java

```diff
@@ -53,10 +59,15 @@ public class Transfer2GPlus {
 
     public static void main(String[] args) throws IOException {
         Path src = Files.createTempFile("src", ".dat");
-        src.toFile().deleteOnExit();
-        byte[] b = createSrcFile(src);
-        testToFileChannel(src, b);
-        testToWritableByteChannel(src, b);
+        File srcFile = src.toFile();
+        srcFile.deleteOnExit();
+        try {
+	        byte[] b = createSrcFile(src);
+	        testToFileChannel(src, b);
+	        testToWritableByteChannel(src, b);
```

Comment:
Please fix the indentation in this file (spaces, not tabs).

Reply:
Updated.